packages/parcel-transformer/src/__tests__/transformer.parceltest.ts

```diff
@@ -121,10 +121,10 @@ it('transforms assets with compiled and extraction babel plugins', async () => {
   expect(extractedCss).toMatchInlineSnapshot(`
     "/* index.jsx generated by @compiled/babel-plugin v0.0.0 */ \\"use strict\\";
     require(\\"compiled-css!?style=._19pk1ul9%7Bmargin-top%3A30px%7D\\");
-    require(\\"compiled-css!?style=._19bvftgi%7Bpadding-left%3A8px%7D\\");
```

Comment:
Updated snapshots to reflect values changed in https://github.com/atlassian-labs/compiled/pull/1266/files#r944090914